html/pfappserver/root/src/views/Configuration/sources/_components/FormTypeLdap.vue

```diff
@@ -95,6 +95,27 @@
       :text="$i18n.t('Realms that will be associated with this source.')"
     />
 
+    <form-group-verify namespace="verify"
+      :column-label="$i18n.t('Verify')"
+      :text="$i18n.t('SSL Verify.')"
+    />
+
+    <form-group-client-cert namespace="client_cert"
+      :column-label="$i18n.t('Client Certificate')"
+    />
+
+    <form-group-client-key namespace="client_key"
+      :column-label="$i18n.t('Client Key')"
+    />
+
+    <form-group-ca-path namespace="ca_path"
+      :column-label="$i18n.t('CA Directory')"
```

Comment:
Add a descriptive help here
It should state that this is only necessary when using certificate authentication against the LDAP directory